### DISCUSSION TITLE
跨年懶人包來拉!!

### DISCUSSION THREAD

**Discussion by spermwhale464** [1101-GS4719-C/Python-Community-GS4719-C#1811] [General]
https://tw.news.yahoo.com/%E6%87%B6%E4%BA%BA%E5%8C%85-2022%E5%85%A8%E5%8F%B0%E8%B7%A8%E5%B9%B4%E6%B4%BB%E5%8B%95-%E7%85%99%E7%81%AB-%E6%BC%94%E5%94%B1%E6%9C%83-%E4%BA%AE%E9%BB%9E-091927782.html
今年好像沒麼亮點，主要是八三夭要在麗寶又要在台北ㄟ，怪怪的
小編我在家裡看電視煙火就好，不想人擠人丫~~

**Comment by Patsy17** [comment]
以前都會跑去現場看演唱會和煙火
不過現在都超懶得去人擠人（長大ㄌ
找朋友一起包棟玩就好～

**Comment by uwje** [comment]
我認為待在家裡比出去人擠人快年來的好，除非有什麼特殊原因，例如跟別人出去，或想看特定的藝人才會讓我願意到現場去觀看。畢竟跨年時都冷冷的，所以待在家裡不僅能禦寒也能同時看到多場跨年的轉播，避免在同一場看到不喜歡的表演，還不能走

**Comment by eric202501** [comment]
對不起跨年剩我在醜 只能在家讀期末 但真的要讀不完了

**Comment by 12Jesse24** [comment]
確實待在家總比外面人擠人來的好，不過今年我們一家人去山上露營，就像隱居的世外高人樣，也是別有一番寧靜呢。

**Comment by B1nn9** [comment]
今年沒特地出門跨年，但在最後3分鐘突然臨時起意，就搭電梯上自家頂樓，結果一倒數完後頂樓四面八方都看得見各地的煙火，其實也算是滿特別的一種感受，最重要的是不會人擠人也能看煙火，看完後也能馬上回家😆
